test/gracefulstop_test.go

```diff
@@ -122,7 +122,8 @@ func (s) TestGracefulStop(t *testing.T) {
 		},
 	}
 	s := grpc.NewServer()
-	testgrpc.RegisterTestServiceServer(s, ss)
+	ss.S = s
```

Comment:
nit: this can be just ss.S = grpc.NewServer() and all the usages can be updated as `ss.S.` like `ss.S.stop()` etc.

Reply:
Done